SEED-platform/seed
# Reorder the metric type drop down text in Program Configuration

#### Any background context you want to provide?

#### What's this PR do?
Revises the wording for the second item in the energy and emission metric type drop down so that the words "Target" and "Actual" are in the same positions and the comparison sign changes.

#### How should this be manually tested?
Go to edit a metric type and confirm that the second option in the drop down is equal to "Target < Actual ...".

#### What are the relevant tickets?
Fixes #3692

#### Screenshots (if appropriate)
